### PR TITLE
DM-21379: Fix validate_drp's HscQuick test script

### PR DESCRIPTION
**Pull request by taranu**
Replaced afwGeom -> geom to minimize annoying warnings and tested every example script.

### CODE REVIEW

File: python/lsst/validate/drp/matchreduce.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def build_matched_dataset(repo, dataIds, matchRadius=None, safeSnr=50.,
     dataIds : `list` of `dict`
         List of `butler` data IDs of Image catalogs to compare to reference.
         The `calexp` cpixel image is needed for the photometric calibration.
-    matchRadius :  afwGeom.Angle(), optional
+    matchRadius :  geom.Angle(), optional
```
Comment (timj) [inline python/lsst/validate/drp/matchreduce.py]: I think docstrings may need the full definition and should also be in back ticks:
```
matchRadius : `lsst.geom.Angle`, optional
```

File: examples/runDecamTest.sh [hunk 2]
```diff
@@ -52,7 +52,7 @@ if [[ $DOPROCESS == true ]]; then
         -i ingestImagesDecam.py
 fi
 
-if [[ $DOVERIFY = true ]]; then
+if [[ $DOVERIFY == true ]]; then
```
Comment (SimonKrughoff) [inline examples/runDecamTest.sh:55]: Interesting.  How did this ever work in the past?

File: python/lsst/validate/drp/matchreduce.py [hunk 3]
```diff
@@ -133,30 +133,32 @@ def build_matched_dataset(repo, dataIds, matchRadius=None, safeSnr=50.,
...
     Returns
     -------
-    catalog_list : afw.table.SourceCatalog
-        List of all of the catalogs
-    matched_catalog : afw.table.GroupView
-        An object of matched catalog.
+    catalog : `lsst.afw.table.SourceCatalog`
+        A new calibrated SourceCatalog.
+    matches : `lsst.afw.table.GroupView`
+        An GroupView of the matched sources.
```
Comment (SimonKrughoff) [inline python/lsst/validate/drp/matchreduce.py]: "An GroupView" --> "A GroupView"?

**Review by SimonKrughoff — APPROVED** [review]
This all looks reasonable.  We should keep an eye on the nightly build when this is merged since that is the only way I know of to test this comprehensively.

File: examples/runDecamTest.sh (hunk 2 above)
Comment (taranu) [reply examples/runDecamTest.sh:55]: Apparently there's no difference between [[$x=true]] and [[$x==true]] in bash.

**Pull request merged**